src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  const a = new Big(parseFloat(numberOne));
+  const b = parseFloat(numberTwo);
+
+  if (operation === 'X') {
+    res = a.times(b);
+  } if (operation === '÷') {
+    res = a.div(b);
+  } if (operation === '+') {
+    res = a.plus(b);
+  } if (operation === '-') {
+    res = a.minus(b);
+  }
```

Comment:
- [ ] This issue hasn't been fixed yet. Kind fix it.
> We still need to add another case for % operation (it turns the number into a percentage by dividing by 100).

Reply:
This function is in calculate.js line 18